inc/classes/class-srm-post-type.php

```diff
@@ -713,11 +713,7 @@ public function srm_autocomplete() {
 
 		$query = get_posts(
 			array(
-				'post_type'      => get_post_types(
-					array(
-						'public' => true,
-					)
-				),
+				'post_type'      => array_filter( get_post_types(), 'is_post_type_viewable' ),
```

Comment:
As mentioned https://github.com/10up/safe-redirect-manager/pull/332#discussion_r1248088127, this will now show the `redirect_rule` post type, which doesn't really make sense in this context. I'd suggest either we `unset` that value or utilize the core `is_post_type_viewable` filter to remove it. Something like

```php
$post_types = array_filter( get_post_types(), 'is_post_type_viewable' );
unset( $post_types[ 'redirect_rule' ] );
```

Reply:
@dkotter Right on! I've got it updated but used `array_filter` to remove that one. Let me know if you'd prefer the `unset` method instead.